src/main/java/org/cyclonedx/maven/DefaultModelConverter.java

```diff
@@ -55,8 +45,18 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.github.packageurl.MalformedPackageURLException;
-import com.github.packageurl.PackageURL;
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Properties;
+import java.util.TreeMap;
+import java.util.stream.Collectors;
```

Comment:
please don't reformat

Reply:
Fixed in 20287587e8123fabf00afdacf6ab1cba581459be.